src/press_quote/quote.macros.html

```diff
@@ -1,11 +1,16 @@
 {#
   * @overview Macros for press quotes.
+  * @module Press Quote
 #}
 
 {% from '../icons/icons.macros.html' import svg %}
 
 {#
   * Renders a single press quote with source logo beneath.
+  * @param {string} Quote text content
+  * @param {string} Associated press logo name
```

Comment:
Perhaps provide a little more information that this is what renders the svg logo